### DISCUSSION TITLE
"OR" operator for issues query

### DISCUSSION THREAD

**Discussion by KaKi87** [community/community#22992] [New to GitHub]
<p>Hello there,</p>
<p>I’m not satisfied of the default GitHub issues home page, so I would like to make a search query for all issues I could be involved in any way, wether they’re open or closed, sorted by last updated date.</p>
<p>Meaning :</p>
<ul>
<li>authored<br>
OR</li>
<li>commented<br>
OR</li>
<li>mentioned<br>
OR</li>
<li>assigned</li>
</ul>
<p>However, I can’t find the “OR” operator, so I would like to know if it’s possible to implement it in any way.</p>
<p>Thanks</p>

**Comment by i-marsh** [comment, marked as answer]
<p>I think this gives what you are looking for… </p>
<p><a href="https://help.github.com/en/articles/searching-issues-and-pull-requests#search-by-a-user-thats-involved-in-an-issue-or-pull-request" rel="nofollow noopener">https://help.github.com/en/articles/searching-issues-and-pull-requests#search-by-a-user-thats-involved-in-an-issue-or-pull-request</a><br>
<code>involves</code> rolls together author, commenter, mentioned, assigned.</p>

**Comment by KaKi87** [comment]
<pre><code>involves:KaKi87 sort:updated-desc
</code></pre>
<p>It works !</p>
<p>Thanks :slight_smile:</p>

**Comment by KaKi87** [comment]
<p>Hello,</p>
<p>Today I have a new situation : I subscribed to an issue created two days ago, and because it’s pretty recent and have nothing to say more about it, I just added a +1 reaction and manually subscribed to it.</p>
<p>So, this issue won’t appear in my list with <code>involves:KaKi87</code> unless I participate to it.</p>
<p>I also looked at the <a href="https://github.com/notifications/subscriptions" rel="nofollow noopener">subscriptions</a> page but it doesn’t allow sorting by updated date.</p>
<p>Thanks</p>
